frontend/src/domains/places/components/PlaceCard/PlaceCard.tsx

```diff
@@ -0,0 +1,66 @@
+import { useState } from 'react';
+
+import Card from '@/@common/components/Card/Card';
+import Flex from '@/@common/components/Flex/Flex';
+import IconButton from '@/@common/components/IconButton/IconButton';
+import Pill from '@/@common/components/Pill/Pill';
+import Text from '@/@common/components/Text/Text';
+import checkIcon from '@/assets/icons/check.svg';
+import editIcon from '@/assets/icons/edit.svg';
+import plusIcon from '@/assets/icons/plus.svg';
+import trashIcon from '@/assets/icons/trash.svg';
+import theme from '@/styles/theme';
+
+import DatePreviewList from '../DatePreviewList/DatePreviewList';
+
+const PlaceCard = () => {
+  const [isClicked, setIsClicked] = useState(false);
+
+  const handleToggle = () => {
+    setIsClicked((prev) => !prev);
+  };
+
+  return (
+    <Card
+      id="경복궁"
+      width="20rem"
+      height="20rem"
+      variant={isClicked ? 'available' : 'default'}
+    >
+      <Flex
+        direction="column"
+        gap={2}
+        justifyContent="flex-start"
+        alignItems="flex-start"
+      >
+        <Flex justifyContent="space-between" width="100%" alignItems="center">
+          {isClicked ? (
+            <IconButton
+              icon={checkIcon}
+              variant="selected"
+              onClick={handleToggle}
+            />
+          ) : (
+            <IconButton
+              icon={plusIcon}
+              variant="select"
+              onClick={handleToggle}
+            />
+          )}
```

Comment:
```suggestion
<IconButton
  icon={isClicked ? checkIcon : plusIcon}
  variant={isClicked ? 'selected' : 'select'}
  onClick={handleToggle}
/>
```

이렇게도 적용할 수 있을 것 같은데 주렁은 어떻게 생각하시나요? 🤔

Reply:
너무 간결하고 좋은 코드네요 🥺